lib/live/form.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require_relative 'view'
+require 'trenni/builder'
+require 'json'
+
+module Live 
+    #Represents a Form within the view layer
+    #Differs from Live::View because the server sends both the 
+    #details and a serialized form from the view layer.
+    class Form < Live::View 
+        
+        #returns [String] javascript code to execute when the form is submitted.
+        def handleForm(details=false)
```

Comment:
We should probably call this 'forward_submit'.

Reply:
[29b8334](https://github.com/socketry/live/pull/6/commits/29b8334b8be5ed1128dc3decfb3f76fcc12a3255)